### PR TITLE
Update release pipeline resource list for Operator nightly

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Update the reference to the nightly release resources in tektoncd/operator

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by nikhil-thomas** [comment]
/hold
until https://github.com/tektoncd/operator/pull/250 merges

File: tekton/resources/nightly-release/overlays/operator/kustomization.yaml [hunk 1]
```diff
@@ -15,4 +15,4 @@ patchesJson6902:
       name: nightly
     path: trigger.yaml
 resources:
-  - github.com/tektoncd/operator/tekton/?ref=main
\ No newline at end of file
+  - github.com/nikhil-thomas/tektoncd-operator/tekton/overlays/nightly-releases?ref=main
```
Comment (vdemeester) [inline tekton/resources/nightly-release/overlays/operator/kustomization.yaml:18]: This should still refer `tektoncd/operator` though 😝

**Review by vdemeester — CHANGES_REQUESTED** [review]

File: tekton/resources/nightly-release/overlays/operator/kustomization.yaml (hunk 1 above)
Comment (nikhil-thomas) [reply tekton/resources/nightly-release/overlays/operator/kustomization.yaml]: i remembered to change the `ref=main` but missed it in the url... 😛 

fixed 🧑‍💻 👍

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Review by afrittoli — COMMENTED** [review]
/approve

**Pull request merged**